app/models/qts_years.rb

```diff
@@ -0,0 +1,29 @@
+# frozen_string_literal: true
+
+# Represents the year a teacher can qualify in
+
+class QtsYears
```

Comment:
Given that we're not going to be instantiating a `QtsYear` instance and this is a collection of class methods, it probably makes more sense as a module and not a class. Also given that it is specific to the student loans policy, how do you feel about namespacing it under the `TslrClaim` model?